.github/workflows/tests.yaml

```
@@ -86,6 +86,10 @@ jobs:
           --exclude tests/.*/*valid_package/*
           src tests
 
+      - name: Mypy
+        run: |
+          mypy --ignore-missing-imports --allow-untyped-decorators --strict src/
```

```suggestion
          mypy --ignore-missing-imports --strict src/
```

The `--allow-untyped-decorators` is not needed here. It is used in Statick for the `deprecated` module.